app/styles/app.scss

```diff
@@ -142,11 +166,14 @@ h2 {
 }
 
 .main_title {
-  background-color: $blue;
-  color:$white;
+  background-color: white;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, -webkit-box-shadow, box-shadow, color, text-align